.github/workflows/ci-workflow.yml

```diff
@@ -3,37 +3,12 @@ name: Continuous testing & Linting
 on: [push]
 
 jobs:
-  test36:
-
-    runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        python-version: [3.6]
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install -e .[dev]
-    - name: Lint with flake8
-      run: |
-        flake8 csp_billing_adapter_amazon
-        flake8 tests
-    - name: Test with pytest
-      run: |
-        pytest --cov=csp_billing_adapter_amazon
-  
   testall:
 
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, "3.10", 3.11]
+        python-version: [3.8, 3.9, "3.10", 3.11, 3.12]
```

Comment:
Note that 3.13 is available in the ubuntu-latest image, if you wanted to expand testing to include 3.13 at this stage, assuming it works.

Reply:
Thanks, noted to add it on the next go around. It should work the same.